cmd/generate/config/rules/gitlab.go

```diff
@@ -22,3 +22,35 @@ func Gitlab() *config.Rule {
 	}
 	return validate(r, tps, nil)
 }
+
+func GitlabPipelineTriggerToken() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "GitLab Pipeline Trigger Token",
+		RuleID:      "gitlab-ptt",
+		Regex:       regexp.MustCompile(`glptt-[0-9a-f]{40}`),
+		Keywords:    []string{"glptt-"},
+	}
+
+	// validate
+	tps := []string{
+		generateSampleSecret("gitlab", "glptt-"+secrets.NewSecret(hex("40"))),
+	}
+	return validate(r, tps, nil)
+}
+
+func GitlabRunnerRegistrationToken() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "GitLab Runner Registration Token",
+		RuleID:      "gitlab-rrt",
```

Comment:
I was on the fence about making this ID a bit more descriptive e.g. `gitlab-runner-registration-token`, but seems to be consistent with the other gitlab ones.